indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,265 @@
+                                </li>
+                            {% endfor %}
+                        </ul>
+                    </div>
+                {% endif %}
+            </div>
+        {% endif %}
+    </div>
+{% endmacro %}
+
+{% macro render_nested_contrib(entry, show_abstract, dont_show_poster_abstract, show_title, show_affiliation, show_length_contribs=false) %}
+    {% set contrib = entry.object if entry.object else entry %}
+    {% set speakers = contrib.speakers %}
+
+    {% if speakers %}
+        <p>
+            <span class="bold">{{ ngettext('Speaker', 'Speakers', speakers|length) }}</span><br />
+            {{ render_names(speakers, show_title=show_title, show_affiliation=show_affiliation) }}
+        </p>
+    {% endif %}
+
+    {% if not contrib.inherit_location and contrib.has_location_info %}
+        <p>
+            <span class="bold">{% trans %}Location{% endtrans %}</span><br />
+            {{ render_place(contrib) }}
+        </p>
+    {% endif %}
+    {% if show_abstract and (not dont_show_poster_abstract or entry.is_poster) and contrib.description %}
+        <div class="description">
+            <span class="bold">{% trans %}Description{% endtrans %}</span>
+            <div>{{ contrib.description }}</div>
+        </div>
+    {% endif %}
+    {% if contrib.subcontributions %}
+        <div>
+            <span class="bold">{% trans %}Subcontributions{% endtrans %}</span>
+            <ul class="sub-contrib-timeline inherited-bg-color-trans inherited-border-color">
+                {% for subchild in contrib.subcontributions %}
+                    <li class="inherited-border-color">
+                        <div class="contrib-heading">
+                            {% if show_length_contribs and not contrib.session.is_poster %}
+                                <span class="timebox inherited-bg-color inherited-txt-color">
+                                    {{ (subchild.duration)|format_human_timedelta(narrow=True) }}
+                                </span>
+                            {% endif %}
+                            <h4>{{ subchild.title }}</h4>
+                        </div>
+                        <div class="contrib-timeline-content">
+                            {{ render_nested_contrib(subchild, show_abstract, dont_show_poster_abstract, show_title, show_affiliation, show_length_contribs) }}
+                        </div>
+                    </li>
+                {% endfor%}
+            </ul>
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_block_date(entry, day, print_date_close_to_sessions) %}
+    {% if print_date_close_to_sessions %}
+        <p class="session-block-date">
+            <span class="bold">{{ day|format_date('EEEE, d MMMM', timezone=entry.event.tz) }}</span>
+            <span class="timebox">
+                {{ format_interval(entry.start_dt.astimezone(entry.event.tzinfo), entry.end_dt.astimezone(entry.event.tzinfo), format='hh:mm') }}
```

Comment:
The signature of `format_interval` is wrong - the `format` parameter is actually a skeleton, so you can just do this:
```suggestion
                {{ format_interval(entry.start_dt.astimezone(entry.event.tzinfo), entry.end_dt.astimezone(entry.event.tzinfo), format='hhmm') }}
```
Though, maybe `HHmm` (hours 0-23) is better here than `hhmm` (hours 1-12)? Not sure how `HHmm` gets formatted for locales that use AM/PM

Reply:
I agree. As for the `HHmm` format for 12-hour locales, it still uses the 24-hour format.